citest/json_contract/observation_verifier.py

```diff
@@ -381,19 +381,75 @@ def export_to_json_snapshot(self, snapshot, entity):
     super(ObservationVerifierBuilder, self).export_to_json_snapshot(
         snapshot, entity)
 
-  def append_verifier(self, verifier, new_term=False):
-    self.append_verifier_builder(
-        _VerifierBuilderWrapper(verifier), new_term=new_term)
+  def EXPECT(self, verifier):
+    """Starts the verifier expression.
 
-  def append_verifier_builder(self, builder, new_term=False):
-    if new_term and self.__current_builder_conjunction:
+    This starts the first verifier clause. Additional verifiers can be added
+    later by using either AND or OR depending on how to combine them.
+
+    Args:
+       verifier: [ObservationVerifier]
+    Returns:
+       self
+    """
+    if self.__current_builder_conjunction:
+      raise ValueError('There are already verifiers added. Use OR or AND')
```

Comment:
expect isnt technically required. The API still has other ways to acquire verifiers.
Requiring EXPECT might be a good idea though. I'll have to rewrite all the tests since it is new.